README.md

```diff
@@ -167,23 +169,48 @@ Use [`func (tf *TF) Cmd(name string, args ...string) *exec.Cmd`](https://pkg.go.
 
 ### Verbose mode
 
-Enable verbose output using the `-v` CLI flag. It works similar to `go test -v`. When enabled, the whole output will be streamed. If disabled, only logs from failed task are send to the output.
+In verbose mode, the whole output will be streamed. If disabled, only logs from failed task are send to the output.
 
-Use [`func (*TF) Verbose`](https://pkg.go.dev/github.com/pellared/taskflow#TF.Verbose) to check if verbose mode was set within the task's command.
+By default, verbose mode is turned on.
+It can be controlled by registering a boolean parameter and assigning it to the field `Taskflow.Verbose`.
+
+This is also provided via the convenience function `taskflow.VerboseParam(*taskflow.Taskflow)`, which needs to be explicitly called.
+The convenience function registers a boolean parameter `verbose`, short `v`, and assigns it to the `Taskflow` field.
+This parameter defaults to `false` by default, so as soon as this parameter is registered, verbose mode has to be explicitly enabled via the CLI.
+This then works similar to `go test -v`. 
+
+You can then also re-use the registered parameter to check if verbose mode was set within the task's command.
 
 ### Default task
 
 Default task can be assigned via `DefaultTask` field in [`type Taskflow`](https://pkg.go.dev/github.com/pellared/taskflow#Taskflow).
 
-When default task is set, then it is run if no task is provied via CLI.
+When default task is set, then it is run if no task is provided via CLI.
 
 ### Parameters
 
-The parameters can be set via CLI using the `key=val` syntax after CLI flags. For example, `go run ./build -v ci=true all` would run the `all` task with `ci` parameter set to `"true"` in verbose mode.
+The parameters can be set via CLI using flag syntax. Flags must be registered in "long form", and have an optional short form using only one letter.
+
+On the CLI, flags can be set in the following ways:
+* `--longName simple` - for simple single-word values
+* `--longName "value with blanks"`
+* `--longName="value with blanks"`
+* `--booleanParam` - setting boolean parameters implicitly to `true`
+* `-l simple` - using short form; Short form also allows assignment via equals.
```

Comment:
```suggestion
* `-l simple` - using short form
```

I removed the ending b/c it is also possible to use quotes + boolean params do not need `true`